gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/CourseGradeOverridePanel.java

```diff
@@ -195,5 +205,36 @@ private String formatPoints(final CourseGrade courseGrade, final Gradebook grade
 		return rval;
 
 	}
+	
+	private String getGradeFromNumber(String newGrade, Map<String, Double> schema)
+	{
+		//Returns the scale whose value equals to the numeric value received, or if it doesn't exists, the highest value lower.
+		Double currentGradeValue = new Double(0.0);
+		try
+		{
+			Double dValue = Double.parseDouble(newGrade.replaceAll(",",".")); //The decimal separator is not relevant here.
```

Comment:
Sorry, @ern , I don't know how to use DecimalFormat to parse a value without knowing the locale and I'm not sure the locale is available here.

And in this case the comma replacement works, because users input is only used to get the closest lower final grade from the scale. It works even if user doesn't know his own locale and uses the wrong decimal separator. Though I admit that is not elegant neither readable.

But if you can share a sample of DecimalFormat ignoring commas, I will do the change.